charts/tidb-lightning/values.yaml

```diff
@@ -90,7 +90,14 @@ annotations: {}
 tolerations: []
 affinity: {}
 
-backend: importer # importer | tidb
+# The delivery backend used to import data.
+# If set to `local`, then the following `sortedKV` should be set.
+backend: importer # importer | local | tidb
+
+# For `local` backend, an extra PV is needed for local KV sorting.
+sortedKV: {}
+#  storageClassName: local-path
```

Comment:
```suggestion
#  storageClassName: local-storage
```

Reply:
To keep consistent with the existing one.

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/3644/commits/b6718ed24a28dd24f770cbed84dbda7b865dd2f2.